src/foundry/client/pixi/perception/color-manager.d.ts

```diff
@@ -0,0 +1,89 @@
+export {};
+
+// TODO: Define Color in common/utils/color.mjs
+type Color = Number;
```

Comment:
I know this is a todo but you probably mean `number`. [`Number`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) probably isn't what you mean.